brainscore_vision/models/resnext101_32x8d_wsl/model.py

```diff
@@ -22,26 +20,8 @@ def get_model(name):
 
 
 def get_layers(name):
-    assert name == 'resnext101_32x8d_wsl'
     return (['conv1'] +
             # note that while relu is used multiple times, by default the last one will overwrite all previous ones
             [f"layer{block + 1}.{unit}.relu"
              for block, block_units in enumerate([3, 4, 23, 3]) for unit in range(block_units)] +
-            ['avgpool'])
-
-
-def get_bibtex(model_identifier):
-    """
-    A method returning the bibtex reference of the requested model as a string.
-    """
-    return """@inproceedings{mahajan2018exploring,
-              title={Exploring the limits of weakly supervised pretraining},
-              author={Mahajan, Dhruv and Girshick, Ross and Ramanathan, Vignesh and He, Kaiming and Paluri, Manohar and Li, Yixuan and Bharambe, Ashwin and Van Der Maaten, Laurens},
-              booktitle={Proceedings of the European conference on computer vision (ECCV)},
-              pages={181--196},
-              year={2018}
-            }"""
```

Comment:
why remove the bibtex?

Reply:
Not sure how this got removed here, fixing in next commit.